### PR TITLE
xset: add page

### PR DESCRIPTION
**Pull request by juligreen**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):** 1.2.4

### CODE REVIEW

File: pages/linux/xset.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# xset
+
+> User preference utility for X.
+> More information: <https://wiki.archlinux.org/title/Display_Power_Management_Signaling>.
```
Comment (navarroaxel) [inline pages/linux/xset.md:4]: We prefer manned as main source
```suggestion
> More information: <https://manned.org/xset>.
```

**Review by CleanMachine1 — APPROVED** [review]

File: pages/linux/xset.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# xset
+
+> User preference utility for X.
```
Comment (sbrl) [inline pages/linux/xset.md:3]: X? Do you mean X11 or the X server?

File: pages/linux/xset.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+> More information: <https://manned.org/xset>.
+
+- Disable screensaver:
```
Comment (sbrl) [inline pages/linux/xset.md:6]: Perhaps this will read better?

```suggestion
- Disable the screensaver:
```

File: pages/linux/xset.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+`xset s off`
+
+- Set screensaver to start after 60 minutes of inactivity:
```
Comment (sbrl) [inline pages/linux/xset.md:10]: Perhaps this will read better?
```suggestion
- Set the screensaver to start after 60 minutes of inactivity:
```

**Review by sbrl — COMMENTED** [review]
Thanks for the new page! I've left some comments below for you to review.

File: pages/linux/xset.md (hunk 2 above)
Comment (juligreen) [reply pages/linux/xset.md:3]: This is copied from the man page.
Isn't X11 just the version of X?
And the X server the actual server implementing the X protocol?
But we can use whatever you want here :)

**Review by CleanMachine1 — APPROVED** [review]

File: pages/linux/xset.md (hunk 2 above)
Comment (sbrl) [reply pages/linux/xset.md:3]: Ah, I see. Let's keep it as-is then.

**Review by sbrl — APPROVED** [review]
Thanks, @juligreen!

**Pull request merged**